src/main/webapp/stats.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Stats</title>
+  <link rel="stylesheet" href="/css/main.css">
+  
+  <script>
+    // Fetch stats and display them in the page.
+    function fetchStats(){
```

Comment:
Run all the javascript through https://prettier.io/playground/ (just copy and paste)